src/client/src/store/config/config/saga.ts

```diff
@@ -78,51 +79,57 @@ export function* loadProjectTypesListSagaAndOptionalFrameworkList(vscode: any) {
       payload: projectType,
     });
 
-    if (projectType === WIZARD_CONTENT_INTERNAL_NAMES.FULL_STACK_APP) {
+    //TODO: probably move and rename this 
+    const rnProjectTypeList = [WIZARD_PROJECT_TYPE.RN_TABBED_APP, WIZARD_PROJECT_TYPE.RN_DRAWER_APP];
```

Comment:
Make sure we change this on #1664 and remove most of the TODOs related to project type